source/collaborate/learn-about-roles.rst

```diff
@@ -89,3 +89,32 @@ Mattermost is designed as a system-of-record, so there isn't an option to delete
     AD/LDAP user accounts can't be deactivated from Mattermost; they must be deactivated from your Active Directory.
 
 
+User ID
+-------
+Users can be specified in Mattermost by username or user ID. Usernames automatically resolve when a match is detected.
+Below are three ways a User's ID can be determined.
+
+Determining a User ID through the System Console (Admin)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Log in to your Mattermost instance as an administrator.
+Go to the System Console by clicking on the gear icon in the top left corner and selecting "System Console."
+In the System Console, navigate to "Users" or a similar section where user management settings are available.
+You should see a list of users, and their IDs should be displayed alongside their usernames.
+Find the user whose ID you want to determine and note it down.
+
+Determining a User ID Using the API
```

Comment:
Please list this as 1 of 2 alternative ways to determine User ID.

Reply:
Done.